PocketKit/Sources/Sync/Slates/Slate+remoteMapping.swift

```diff
@@ -3,10 +3,24 @@ import Foundation
 
 extension Slate {
     typealias Remote = SlateParts
+    typealias GetSlateLineupRemote = GetSlateLineupQuery.Data.GetSlateLineup.Slate
```

Comment:
This bit shouldn't be necessary, you can access the `SlateParts` fragment from the `GetSlateLineupQuery.Data.GetSlateLineup.Slate` object and initialize using the fragment instead. (that way it doesn't matter what query we're using)